### PR TITLE
Fix 404 page

### PR DESCRIPTION
**Pull request by makkes**
The page isn't rendered properly (completely empty) with that comment at the top.

Before: https://fluxcd.io/does-not-exist

After: https://deploy-preview-1551--fluxcd.netlify.app/does-not-exist

### CODE REVIEW

**Comment by hiddeco** [comment]
In the preview the search doesn’t appear to actually work for me.

**Comment by makkes** [comment]
> In the preview the search doesn’t appear to actually work for me.

Oh right. I already know why. On it.

File: assets/scss/_page.scss [hunk 1]
```diff
@@ -24,6 +24,12 @@
 }
 
 .page404 {
+
```
Comment (hiddeco) [inline assets/scss/_page.scss:27]:
```suggestion
```

**Review by hiddeco — APPROVED** [review]

**Comment by hiddeco** [comment]
Nice work @makkes! :bow:

**Pull request merged**